### DISCUSSION TITLE
Because the percentage of successful points in the last 24 hours is getting lower and lower?

### DISCUSSION THREAD

**Discussion by albegme** [Chia-Network/chia-blockchain#9776] [Support]
I see that every day the percentage number of my success points is low and not as before, which was close to 100%, and I am curious that the MiB of node connection load is 0 when before it always increases with the value of download. Could you tell me if it's a software or hardware problem?
![image](https://user-images.githubusercontent.com/97495100/148879371-075ab188-f91c-470b-876c-50fc3298f8e1.png)
![image](https://user-images.githubusercontent.com/97495100/148879484-59e9f39c-1b7c-4eb7-9f3b-da374728654b.png)

**Comment by smellsgood-tech** [comment]
i have same problem and i did this:

1 ~\daemon\chia.exe plots check 
   if any one or two bad plot just remove bad plot, if more than two bad plot just change disk 
2 port forwarding port 8444 to router
3 set priority in task manager ( set to high priority)

to day i got 96% succes point, , I hope it can be useful for others

**Comment by hajes** [comment]
check out logs, there is usually duplicate, incomplete plots.
